### ISSUE TITLE
logs not writing

### ISSUE THREAD

**Issue by paramitanit** [jcabi/jcabi-aspects#246]
Hi @yegor256 

i have problem, logs are not writing for me too. i used the class files, pom.xml as the given URL(https://github.com/jcabi/jcabi-aspects/issues/179) but the pom xml showing error.I m attaching the error and also the pom.xml file..please help me asap
[pom.txt](https://github.com/jcabi/jcabi-aspects/files/1222735/pom.txt)
![pom](https://user-images.githubusercontent.com/30828893/29280211-f7ef8a00-8137-11e7-99e5-cdd39056a76a.png)

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by yegor256** [comment]
@paramitanit they won't work in the IDE, you know that?

**Comment by paramitanit** [comment]
@yegor256 yeah sir...the file jcabi.log is not generating: c//log//jcabi.log in properties file 

log4j.rootLogger=INFO, file, CONSOLE
log4j.appender.file=org.apache.log4j.RollingFileAppender
log4j.appender.file.File=C:\\log\\jcabi.log
log4j.appender.file.MaxFileSize=10MB
log4j.appender.file.MaxBackupIndex=1
log4j.appender.file.layout=org.apache.log4j.PatternLayout
log4j.appender.file.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n

log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
log4j.appender.CONSOLE.layout=com.jcabi.log.MulticolorLayout
log4j.appender.CONSOLE.layout.ConversionPattern=[%color{%-5p}] %c: %m%n

**Comment by paramitanit** [comment]
@yegor256 i m using eclipse indigo

**Comment by yegor256** [comment]
@paramitanit jcabi-log won't work if you run your app from Eclipse. Run it from Maven.
